ddd-koans/Value Object Pt. 2 & Entity/Length and tempo/task.html

```diff
@@ -0,0 +1,59 @@
+<html>
+<q>
+    Let's play this tune a little bit faster - says conductor.
+</q>
+</br>
+<q>
+    Like what? - asks drummer
+</q>
+</br>
+<q>
+    10 BPM more.
+</q>
+</br>
+<q>
+    Ok, so it gonna be 130BPM!
+</q>
+
+<h3>How to measure the real length of note value</h3>
```

Comment:
I would add some example of how the calculation is made in real life like some equation to illustrate what needs to be implemented according to Tempo and NoteValues